File: algorithm/boj/_1000/1065.py
import sys

if __name__ == "__main__":
    n = int(sys.stdin.readline())
    
    if n<100:
        print(n)
    elif n==1000:
        print('144')
    else:
        cnt = 99
        for i in range(100, n+1):
            if i//100 + i%10 == 2*(i//10%10):
                cnt += 1
        print(cnt)

File: algorithm/boj/_1000/1065.c
#include <stdio.h>

int main() {
    int n;
    int cnt;
    scanf("%d", &n);
    
    if(n<100) {
        printf("%d\n", n);
    } else if(n==1000) {
        printf("144\n");
    } else {
        cnt = 99;
        for(int i=100; i<n+1; i++) {
            if(i/100+i%10 == 2*(i/10%10))
                cnt += 1;
        }
        printf("%d\n", cnt);
    }
    return 0;
}